### ISSUE TITLE
Action bar has extra padding on top without status bar showing

### ISSUE THREAD

**Issue by hamidbsd** [NativeScript/NativeScript#8081]
**Environment**
{N} 6.2

**Describe the bug**
Action bar always has extra padding on top even without status bar

**To Reproduce**
Create a Blank Angular&TypeScript app in Sidekick and run on an Android phone.
When holding the menu button on the phone, the app window becomes half height and status bar disappears and the Action bar remains with the extra padding on top.

**Expected behavior**
Actionbar to adjust in respect to the presence of the status bar.

**Sample project**
Blank template app of Angular&Typescript

**Comment by hamidbsd** [comment]
This might be because of this new feature: https://github.com/NativeScript/NativeScript/pull/7805
There is no problem in {N} 6.1.2 and works as expected.

**Comment by bundyo** [comment]
Not sure if #7805 is connected as [setContentInsetsAbsolute](https://developer.android.com/reference/android/widget/Toolbar.html#setContentInsetsAbsolute(int,%20int)) only modifies left and right Toolbar content insets. Can you check if changing androidContentInset property does something for this padding?

Btw, are you using the {N} Theme and which version?

**Comment by iamabs2001** [comment]
yes i removed status bar but still its look like taking space of status bar, i tried to set actionbar height but not working
